website/views/shared/profiles/achievements.jade

```diff
@@ -1,5 +1,8 @@
 mixin simpleAchiev(achiev)
-  div(ng-init='earnedClass = achiev.earned ? "" : "muted"')&attributes(attributes)
+  div(ng-init='earnedClass = achiev.earned ? "" : "muted"', 
+      data-popover-html='<div class="{{earnedClass}}">{{achiev.title}}<hr>{{achiev.text}}</div>',
```

Comment:
Clicked and mouseenter popover

![habitica__achievements_-_google_chrome_2016-11-07_18-49-03](https://cloud.githubusercontent.com/assets/842273/20068955/e4e23ff6-a51a-11e6-9a57-d78108dda224.png)